src/app/coursegrab/controllers/update_notification_controller.py

```diff
@@ -13,5 +13,7 @@ def get_methods(self):
     def content(self, **kwargs):
         data = request.get_json()
         user = kwargs.get("user")
+
+        # Cast "NONE" string to null value
```

Comment:
might be worth specifying that `NONE` is a constant coming from `constants.py`, the name could be a bit confusing for python newcomers (`None` vs `NONE`)